src/render.jl

```diff
@@ -137,10 +137,12 @@ The documentation will be available from
 """
 function save(file::String, modulename::Module; args...)
     config = Config(; args...)
-    mime = MIME("text/$(strip(last(splitext(file)), '.'))")
-    save(file, mime, documentation(modulename), config)
+    strip(last(splitext(file)), '.') == "html"              ?
+        savehtml(file, documentation(modulename), config) :
+        savemd(file, documentation(modulename), config)
 end
```

Comment:
This won't scale well with any additional future output formats. Multiple dispatch is a positive here. Let's stick to the generic `save`.